src/qinfer/hyper_heuristic_optimisers.py

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
We should make sure the copyright header and ``-*- coding`` lines are kept on this file as well.